lib/result-view.coffee

```diff
@@ -130,14 +130,15 @@ class ResultView
                 @element.classList.add 'rich'
                 @setMultiline true
 
+            container.appendChild htmlElement
+
             if mimeType is 'text/html'
-                container.classList.add('html')
+                if @getAllText() isnt ''
+                    @element.classList.remove 'rich'
```

Comment:
Wait, better put the comment above the line with  `container.appendChild htmlElement`